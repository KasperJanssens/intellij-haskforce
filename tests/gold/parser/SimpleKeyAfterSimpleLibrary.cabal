library
    js-sources:  onejs, twojs

name: koekoek