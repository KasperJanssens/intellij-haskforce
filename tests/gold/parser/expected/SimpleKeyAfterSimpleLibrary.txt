Cabal File
  KEY_OR_CONFIG
    CONFIG
      LIBRARY
        PsiElement(LIBRARY)('library')
        LIBRARY_KEYS
          BUILD_INFORMATION
            PsiElement(CabalTokenType.js-sources)('js-sources')
            PsiElement(CabalTokenType.:)(':')
            FILE_NAME
              PsiElement(CabalTokenType.varidRegexp)('onejs')
            PsiElement(CabalTokenType.,)(',')
            FILE_NAME
              PsiElement(CabalTokenType.varidRegexp)('twojs')
  KEY_OR_CONFIG
    KEY
      PsiElement(CabalTokenType.name)('name')
      PsiElement(CabalTokenType.:)(':')
      VARID
        PsiElement(CabalTokenType.varidRegexp)('koekoek')